bats/system-tables.bats

```diff
@@ -116,6 +116,16 @@ teardown() {
     [ "${#lines[@]}" -eq 5 ]
 }
 
+@test "query dolt_diff_ system table without committing table" {
+    dolt sql -q "create table test (pk int not null primary key);"
+    dolt sql -q "insert into test values (0), (1);"
+    run dolt sql -q 'select * from dolt_diff_test;'
+    [ $status -eq 0 ]
+    [[ "${lines[1]}" =~ "| to_pk | to_commit | from_pk | from_commit | diff_type |" ]] || false
```

Comment:
Maybe change these to exact equals not regexes?

`[ "${lines[3]}" =~ "| 0     | HEAD      | <NULL>  | current     | added     |" ] `